src/utils/__tests__/storybook-utils.test.ts

```diff
@@ -0,0 +1,23 @@
+import { getStoryId } from "@/utils";
+import { HashEntry } from "@storybook/manager-api";
+import { componentEntry, storyEntry } from "./storybook-utils.test.helpers";
+
+describe("id-utils", () => {
+  describe("getStoryId", () => {
+    it("should return the resolvedEntry first child when the type is 'component'", () => {
+      const result = getStoryId(componentEntry);
```

Comment:
I'm wondering if there's a better way to identify the playground story

Reply:
Maybe using story.parameters?

Reply:
I have to know the name even before story load, parameters are loaded only on first visit in a story

Reply:
I pretty much checked the entire api and the entire flow, also according to source code of storybook's official addons, couldn't come up with a better naming, it's the best available for our needs (to know before playground story loaded)